src/gui/qgsscalewidget.h

```diff
@@ -32,6 +32,8 @@ class GUI_EXPORT QgsScaleWidget : public QWidget
 {
     Q_OBJECT
     Q_PROPERTY( bool showCurrentScaleButton READ showCurrentScaleButton WRITE setShowCurrentScaleButton )
+    Q_PROPERTY( bool scale READ scale WRITE setScale )
```

Comment:
I'm not sure if it helps anywhere else but for QML bindings - and that's not really a criteria for QWidgets. But it won't hurt either :)